requirements.txt

```diff
@@ -1,11 +1,9 @@
-discord
-praw
-bs4
-lxml
-google-api-python-client
-oauth2client
-python-fotmob-wrapper
-datetime
-tabulate
-Pillow
-pandas
+discord~=1.7.3
+bs4~=0.0.1
+datetime~=4.3
+tabulate~=0.8.9
+Pillow~=9.0.0
+pandas~=1.4.0
+
+requests~=2.27.1
+beautifulsoup4~=4.10.0
```

Comment:
Is there a reason we've decided to specify versions?

Reply:
General hygiene for dependency management. This will ensure build are easily reproducible in any environment, and predictable.

Reply:
Good Answer. ⭐